src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -855,10 +855,10 @@ def forward(
 
         device = input_ids.device if input_ids is not None else inputs_embeds.device
 
-        visual_shape = list(input_shape)
+        visual_shape = list(torch.empty(size=input_shape).size())
```

Comment:
What's the reason for this?

Reply:
![image](https://user-images.githubusercontent.com/31131753/144010018-85d79b79-29fb-4c80-896f-49b7756f2eb5.png)
![image](https://user-images.githubusercontent.com/31131753/144010067-82eb25d0-9185-4320-be31-6fce938b0d96.png)

Reply:
Yes @linchpinlin exactly. Thanks
 @michaelbenayoun  I get tensor size error if I don't apply these changes